Source/WebKit/Shared/CoordinatedGraphics/CoordinatedGraphicsScene.h

```diff
@@ -70,6 +71,10 @@ class CoordinatedGraphicsScene : public ThreadSafeRefCounted<CoordinatedGraphics
     bool isActive() const { return m_isActive; }
     void setActive(bool active) { m_isActive = active; }
 
+    const Vector<WebCore::IntRect>& lastDamagedRects() const { return m_lastDamagedRectsAreUnreliable ? m_noDamagedRects : m_lastDamagedRects; }
```

Comment:
It should be fine to return an empty `Vector`. To make the compiler not complain about mismatched types, we can write:

```suggestion
    const Vector<WebCore::IntRect>& lastDamagedRects() const {
       if (m_lastDamagedRectsAreUnreliable) {
           static const Vector<WebCore::IntRect> empty;
           return empty;
       }
       return m_lastDamagedRects;
   }
```

Reply:
done